elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLQueryEngine.java

```diff
@@ -60,13 +61,14 @@
 @Slf4j
 public class SQLQueryEngine extends QueryEngine {
     private final EntityManagerFactory entityManagerFactory;
-
+    private final TransactionCancel transactionCancel;
```

Comment:
I don't see the implementation of executeQuery that returns a Future<QueryResult> here.